client/signup/steps/plans/index.jsx

```diff
@@ -82,7 +82,11 @@ module.exports = React.createClass( {
 	},
 
 	isFreeTrialFlow: function() {
```

Comment:
What do you think about renaming this now that it returns true even if the current flow isn't `free-trial`? `areFreeTrialsEnabled()` or something.